components/files/configurationmanager.cpp

```diff
@@ -164,6 +164,10 @@ bool ConfigurationManager::loadConfig(const boost::filesystem::path& path,
     return false;
 }
 
+const int escape_hash_filter::sEscape = '@';
+const int escape_hash_filter::sEscapeIdentifier = 'a';
+const int escape_hash_filter::sHashIdentifier = 'h';
```

Comment:
This is the fix for the build error. Have a look at the accepted answer here:

http://stackoverflow.com/questions/5391973/undefined-reference-to-static-const-int